addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,55 @@
+# iCloud Binding
+
+The iCloud Binding helps to integrate apple device information into openHAB.
+
+## Supported Things
+
+The following devices are known to work with this binding:
+* iPhone 6s
+* iPhone 5c
+* iPhone 7
+* iPad Air 2/2017/Pro
+
+But other devices should work as well. Please provide feedback if you have tested another device type. We will add it to the list.
+
+## Discovery
+
+For each iCloud account to be included in openHAB a binding (bridge) needs to be configured with your apple id, password & refresh rate. 
```

Comment:
why "binding (bridge)"?

"Apple ID"?

& -> and

is refresh rate optional? Feels like it should be...

Reply:
Fortunately I found a volunteer to create the documentation :-) ... the refresh rate is mandatory (to keep the position data up-to-date).

Reply:
Changed to "id" and replaced "&" with "and" - I´ll commit it later today.